### ISSUE TITLE
npm start on windows errors

### ISSUE THREAD

**Issue by habibmokni** [StartBootstrap/pro-feedback#3]
Hey, I'm using Windows 10, I'm getting this error while running `npm install` : 

![err1](https://user-images.githubusercontent.com/15965492/75630262-6a785500-5be9-11ea-804d-4efbe0a807df.PNG)

and this error after `npm start` : 

![err](https://user-images.githubusercontent.com/15965492/75630105-17ea6900-5be8-11ea-9334-041be0f08495.PNG)

**Comment by initplatform** [comment]
Hello.

It looks like you are using the windows command prompt instead of a unix style shell. 

We recommend either Git Bash or Windows Subsystem for Linux

More info and links to both can be found here: <https://docs.startbootstrap.com/sb-admin-pro-angular/quickstart#unix-style-shell>

All the best!

**Comment by habibmokni** [comment]
> Hello.
> 
> It looks like you are using the windows command prompt instead of a unix style shell.
> 
> We recommend either Git Bash or Windows Subsystem for Linux
> 
> More info and links to both can be found here: https://docs.startbootstrap.com/sb-admin-pro-angular/quickstart#unix-style-shell
> 
> All the best!

I'm facing the same problem with GitBash too.
But when I run just `ng serve` it's working fine on port 4200 without problem but not with `npm start`!

**Comment by initplatform** [comment]
Found the source of the problem. 

## tldr; 

We just released builds:

- sb-admin-pro-angular 1.1.2
- sb-ui-kit-pro-angular 1.0.2

 fixing the issue, which you can download from the [Start Bootstrap shop](https://shop.startbootstrap.com/my-account/) 

of you can fix it locally by doing:

```bash
npm install -D cross-env
```
 and then changing the `ng` script in `package.json` from:

```json
"ng": "node --max_old_space_size=2048 ./node_modules/.bin/ng",
```

to: 

```json
"ng": "cross-env NODE_OPTIONS=--max_old_space_size=2048 ./node_modules/.bin/ng",
```

## Explanation 
On Windows, when you install angular, the file in `node_modules/.bin/ng` is actually a shell script that determines your environment and then calls the node script in `node_modules/@angular/cli/bin/ng`. 

On unix systems, `node_modules/.bin/ng` is a javascript file that can be called directly with node. 

The reason we call ng with node, is so that we can set `max_old_space_size=2048` for certain flavors of linux that don't allocate enough memory for node. 

So the solution is to use NODE_OPTIONS instead to set max_old_space_size, which means we need to bring in a new dependency `cross-env` in order to set environment variables in all platforms from npm scripts. 

@habibmokni Thanks for filing the bug and your support!

**Comment by habibmokni** [comment]
Works perfectly! Thanks for the help and the explanation
